projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -5961,4 +5961,18 @@ export class IgxGridBaseDirective extends DisplayDensityBase implements
             advancedFilteringDialog.closeDialog();
         }
     }
+
+    /**
+     * @hidden
+     * @internal
+     */
+    public get getScrollbarSize() {
```

Comment:
This is named as a method but is actually a getter. Can be named more descriptively - pinStartSize or similar.
Also check can be simplified to isPinningToStart ? pinnedWidth : headerFeaturesWidth